src/core_modules/capture-core/metaDataMemoryStoreBuilders/programs/factory/transformNodeFuntions/transformNodeFunctions.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+import React from 'react';
+
+export const transformTrackerNode = (node: Object, index: number, nodeToElementFn: (...args : Array<any>) => any) => {
```

Comment:
NIT: Flow declaration part3. `nodeToElementFn` can be more specific I think. We can have a chat about these flow declarations, but I think it's a good exercise to look into this a bit.